packages/blitz/src/cli/utils/check-latest-version.ts

```diff
@@ -23,153 +21,63 @@ function getUpdateString(packageName: string, tag: string, isGlobal?: boolean) {
   }
 }
 
-async function findNodeModulesRoot(src: string) {
-  const blitzPkgLocation = dirname(
-    (await findUp("package.json", {
-      cwd: resolveFrom(src, "blitz"),
-    })) ?? "",
-  )
-
-  if (!blitzPkgLocation) {
-    throw new Error("Internal Blitz Error: unable to find 'blitz' package location")
-  }
-
-  return blitzPkgLocation.includes(".pnpm")
-    ? join(blitzPkgLocation, "../../../../")
-    : join(blitzPkgLocation, "../")
-}
-
 export async function checkLatestVersion() {
   if (!isInternalBlitzMonorepoDevelopment) {
     const fetch = await import("node-fetch")
     const boxen = await import("boxen")
     const versions = readVersions()
-    const nodeModulesRoot = await findNodeModulesRoot(process.cwd())
-    const dotBlitzCacheExists = fs.existsSync(
-      join(nodeModulesRoot, ".blitz", "checkUpdateCache.json"),
-    )
-    let dotBlitzCache
-    let shouldRun = true
+    const versionsArray = Object.entries(versions)
 
-    if (dotBlitzCacheExists) {
-      dotBlitzCache = fs.readFileSync(join(nodeModulesRoot, ".blitz", "checkUpdateCache.json"))
-      const now = new Date()
-      const msBetweenTimes = Math.abs(
-        superjson.parse<{lastUpdated: Date}>(dotBlitzCache.toString()).lastUpdated.getTime() -
-          now.getTime(),
-      )
-      const hoursBetweenTimes = msBetweenTimes / (60 * 60 * 1000)
-      shouldRun = hoursBetweenTimes > 24
-    }
+    let shouldRun = true
 
     if (shouldRun) {
       let errors: {message: string; instructions: string}[] = []
       try {
         const blitzResponse = await fetch.default(returnNpmEndpoint("blitz"))
         const remoteBlitzVersions = (await blitzResponse.json()) as Record<string, string>
 
-        const blitzNextResponse = await fetch.default(returnNpmEndpoint("@blitzjs/next"))
-        const remoteBlitzNextVersions = (await blitzNextResponse.json()) as Record<string, string>
-
-        const blitzAuthResponse = await fetch.default(returnNpmEndpoint("@blitzjs/auth"))
-        const remoteBlitzAuthVersions = (await blitzAuthResponse.json()) as Record<string, string>
-
-        const blitzRpcResponse = await fetch.default(returnNpmEndpoint("@blitzjs/rpc"))
-        const remoteBlitzRpcVersions = (await blitzRpcResponse.json()) as Record<string, string>
+        for (const version of versionsArray) {
+          const versionType: string = version[0]
+          const versionValue: typeof version[1] = version[1]
```

Comment:
This is inferred automatically, you don't have to type it manually. 
```suggestion
          const versionValue = version[1]
```

Reply:
Ah thanks